controllers/slice/reconciler.go

```diff
@@ -56,9 +56,9 @@ type SliceReconciler struct {
 	WorkerNetOpClient  WorkerNetOpClientProvider
 }
 
-//+kubebuilder:rbac:groups=networking.kubeslice.io,resources=slice,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=networking.kubeslice.io,resources=slice/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups=networking.kubeslice.io,resources=slice/finalizers,verbs=update
+//+kubebuilder:rbac:groups=networking.kubeslice.io,resources=slices,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Do we need to change this? resource name is still `slice`. Only plural name is changed

Reply:
Hi @kmjayadeep operator is giving error if we have resource name as `slice` because if we give resource name as slice and run `make manifests` below clusterrole config got added which is having resource name as `slice`. 
```
- apiGroups:
  - networking.kubeslice.io
  resources:
  - slice
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
```
above config will not be able to identify our custom resource which is `slices.networking.kubeslice.io` because above config will try to find out `slice.networking.kubeslice.io`